docs/README.md

```diff
@@ -1,49 +1,61 @@
-📢 Use this project, [contribute](https://github.com/vtex-apps/CHANGEME) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+📢 Use this project, [contribute](https://github.com/vtex-apps/elfsight) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-# APP NAME
+# Elfsight
 
 <!-- DOCS-IGNORE:start -->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 <!-- DOCS-IGNORE:end -->
 
-Under the app's name, you should explain the topic, giving a **brief description** of the **app's functionality** (what is it for?) in a store.
+Elfsight widgets help 300,000+ website owners to increase sales, engage visitors, collect leads and more.
 
-Next, you can **add media** (either an image of a GIF) if possible, so that users can better understand how the app works in practice. 
+![image](https://user-images.githubusercontent.com/284515/87573824-dbee0700-c6a3-11ea-9ca2-0f03809b785b.png)
 
-![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)
+## Configuration 
 
-## Configuration
+1. Adding the app as a theme dependency in the `manifest.json` file;
 
-It is possible to install in your store either by using App Store or the VTEX IO Toolbelt.
+```json
+"dependencies": {
+  "vtex.elfsight": "1.x"
+}
+```
 
-### Using VTEX App Store
+2. Add the block `elfsight` to your theme `blocks.json` passing the prop `appId`.
```

Comment:
where should it be added to? Is there a specific page template?

Reply:
It can be added anywhere